pallets/pips/src/benchmarking.rs

```diff
@@ -137,24 +137,21 @@ fn pips_and_votes_setup<T: Trait>(
     Ok((origin, did))
 }
 
-fn enact_call<T: Trait>(num_approves: u32, num_rejects: u32, num_skips: u32) -> Call<T> {
+fn enact_call<T: Trait>(num_approves: usize, num_rejects: usize, num_skips: usize) -> Call<T> {
     let seed = [42; 32];
     let mut rng = ChaCha20Rng::from_seed(seed);
-    let mut indices: Vec<u32> = (0..(num_approves + num_rejects + num_skips)).collect();
-    indices.shuffle(&mut rng);
+    let mut snapshot_results: Vec<_> = iter::repeat(SnapshotResult::Approve)
+        .take(num_approves)
+        .collect();
+    snapshot_results.extend(iter::repeat(SnapshotResult::Reject).take(num_rejects));
+    snapshot_results.extend(iter::repeat(SnapshotResult::Skip).take(num_skips));
+    snapshot_results.shuffle(&mut rng);
```

Comment:
Could use `.chain(...)` later for cleanup when you've got the semantics nailed down.